week-5/LosIncreibles/data-extraction.rb

```diff
@@ -0,0 +1,66 @@
+require 'csv'
+
+class Array
+    def getId(param)
+        self.index(param.gsub(/\'/,"''")) + 1
+    end
+end
+
+class String
+    def scapeColon
+        self.gsub(/\'/,"''")
+    end
+end
+
+
+customers = CSV.read('SalesJan2009.csv')
+
+fileCustomers=File.open("customers.sql","w")
+fileSales=File.open("sales.sql","w")
+fileBusiness=File.open("business.sql","w")
+
+businessArray = customers.map{|row| row[5..6]}.uniq[1..-1]
+customersArray = customers.map{|row| row[7..10]}.uniq[1..-1]
+
+# p customersArray
+
+
+businessArray.each do |row|
+    row.map! {|registro| registro.gsub(/\'/,"''")}
+    fileBusiness.puts "insert into business values(default,'#{row[0]}','#{row[1]}');"
+end
+
+businessNamesArray = businessArray.map {|b| b[0]  }
+# p businessNamesArray.getId('Mothers with Children')
+
+productArray = ['Caribbean Tour','Inka trail','Romantic Paris']
+# p productArray.getId('Caribbean Tour')
+
+posArray = ['Facebook','Shop','Web','Whatsapp']
+# p posArray.getId('Facebook')
+
+paymentArray = ['Amex','Diners','Mastercard','Visa']
+# p paymentArray.getId('Diners')
+
+
+customers = customers[1..-1]
```

Comment:
What does this line do?

Reply:
It returns all the elements of the array but not the first one.